core/authority_discovery/publisher/address_publisher.cpp

```diff
@@ -122,7 +122,7 @@ namespace kagome::authority_discovery {
         authorities,
         authority_discovery_api_->authorities(block_tree_->bestBlock().hash));
 
-    auto audi_key = keys_->getAudiKeyPair(authorities);
+    auto audi_key = keys_->getAudiKeyPair();
```

Comment:
Validator must sign and publish using one active audi key (intersect runtime and keystore), or use all keys (in loop)